### PR TITLE
Patch for PR of OS min/max versions

### PR DESCRIPTION
**Pull request by DIOHz0r**
### Changes description
Add missed code for the PR #509

### CODE REVIEW

File: install/policies/mdm.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
       'type_data'                           => '',
       'unicity'                             => 1,
       'plugin_flyvemdm_policycategories_id' => $category,
-      'comment'                             => __('Use TLS.', 'flyvemdm'),
+      'comment'                             => __('Use TLS', 'flyvemdm').'.',
```
Comment (btry) [inline install/policies/mdm.php]: put the dot in the localizable string

File: install/upgrade/update_to_dev.php [hunk 2]
```diff
@@ -163,6 +163,12 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addField($table, 'apple_max_version', 'bool', ['after' => 'apple_min_version']);
    $migration->addKey($table, 'group', 'group');
    $migration->addKey($table, 'plugin_flyvemdm_policycategories_id', 'plugin_flyvemdm_policycategories_id');
+   if ($DB->fieldExists($table, 'is_android_policy')) {
+      $migration->dropField($table, 'is_android_policy');
+   }
+   if ($DB->fieldExists($table, 'is_apple_policy')) {
+      $migration->dropField($table, 'is_apple_policy');
+   }
```
Comment (btry) [inline install/upgrade/update_to_dev.php]: Migration::dropField() already checks if the field exists before actually dropping it. This change is useless

File: install/policies/mdm.php (hunk 1 above)
Comment (DIOHz0r) [reply install/policies/mdm.php]: Why? the dot it's at the end and we could use the same translation of the upper line or do you really want to have to translate two string just for only one dot?

**Comment by ajsb85** [comment]
Hi, @DIOHz0r 
`fix(install): fix for db schema update and locale`
Should be:
`fix(install): db schema update and locale`
Thanks

**Comment by Naylin15** [comment]
Fixing issues that complement this PR and making several tests in the server

I think I found the solution but I need to test it some more

One of the main issues was that the DB was not being updated properly, the values of the os versions were not being added when running cli_install.php

**Comment by btry** [comment]
Indeed. I'll point to you the code that you need to update.

**Comment by btry** [comment]
@Naylin15 have a look into the PluginFlyvemdmInstaller::createPolicies()

This method is in charge of creating / updating the policies in the database. You need to modify it to take into account the new columns you introduced. Once done, the update process should run as you expected.

File: install/policies/mdm.php (hunk 1 above)
Comment (btry) [reply install/policies/mdm.php]: Maybe in some languages the punctuation may be very different to latin languages. We cannot do assumption about locales.

Moreover this policy willl change its name see #494 We wil need to improve comment and probably use longer text than only copying the name.

**Pull request merged**